app/api/v1/endpoints/upload.py

```
@@ -0,0 +1,40 @@
+    container_client = blob_service_client.get_container_client("컨테이너 이름")  # get_container_client 사용
+
+    try:
+        # 파일 이름 설정
+        file_name = file.filename
+
+        # Azure Blob Storage에 파일 업로드
+        blob_client = container_client.get_blob_client(blob=file_name)
+        blob_client.upload_blob(file.file)
+
+        # 업로드된 Blob의 URL 생성
+        blob_url = f"https://{blob_service_client.account_name}.blob.core.windows.net/{CONTAINER_NAME}/{file_name}"
```

Ruff (0.8.2)

Undefined name `CONTAINER_NAME`

(F821)